src/api-mocks/submissions.js

```diff
@@ -0,0 +1,30 @@
+import {BASE_URL, getDefaultFactory} from './base';
+
+// FIXME - this is incomplete, the prop types aren't detailed enough.
+const SUBMISSION_DETAILS = {
+  id: '458b29ae-5baa-4132-a0d7-8c7071b8152a',
+  url: `${BASE_URL}submissions/458b29ae-5baa-4132-a0d7-8c7071b8152a`,
+  form: `${BASE_URL}forms/mock`,
+  steps: [
+    {
+      id: '6ca342af-86c7-451c-a19f-65050b2eee5c',
+      name: 'Step 1',
+      url: `${BASE_URL}submissions/458b29ae-5baa-4132-a0d7-8c7071b8152a/steps/9e6eb3c5-e5a4-4abf-b64a-73d3243f2bf5`,
+      formStep: `${BASE_URL}forms/mock/steps/9e6eb3c5-e5a4-4abf-b64a-73d3243f2bf5`,
+      isApplicable: true,
+      completed: false,
+      canSubmit: true,
+    },
+  ],
+  submissionAllowed: 'yes',
+  isAuthenticated: false,
+};
+
+/**
+ * Return a submission object as if it would be returned from the backend API.
+ * @param  {Object} overrides Key-value mapping with overrides from the defaults. These
+ *                            are deep-assigned via lodash.set to the defaults, so use
+ *                            '.'-joined strings as keys for deep paths.
+ * @return {Object}           A submission object.
+ */
+export const getSubmission = getDefaultFactory(SUBMISSION_DETAILS);
```

Comment:
```suggestion
export const makeSubmission = defaultFactory(SUBMISSION_DETAILS);
```

Or "build" to mirror FactoryBoy and convey that there's no persistency.  Same for the other factory functions.  Then there's no clash with the GET from the http method.

Reply:
good suggestion, I'll adapt the code!